File: JAVASCRIPT PROGRAMMING/Test60.js
"use strict";
// 60. PRINT THE DIGITS OF A NUMBER (INPUT: 153 OUTPUT: 1 5 3)
const digitNumber = (input) => {
    let temp = 1;
    while (Math.floor(input / temp) >= 10) {
        temp *= 10;
    }
    while (temp > 0) {
        process.stdout.write(`${Math.floor(input / temp)} `);
        input %= temp;
        temp = Math.floor(temp / 10);
    }
    console.log();
};
digitNumber(153);
digitNumber(1234);
digitNumber(12345);

File: PYTHON PROGRAMMING/Test60.py
# 60. STRONG NUMBER
from math import *
def Strong(num):
    temp = 1
    NUM = num
    count = 0
    while num != 0:
        rem = num % 10
        for i in range (1, rem + 1):
            temp *= i
        count += temp
        temp = 1
        num = floor(num / 10)
    if NUM == count:
        print(f"{NUM} Is A Strong Number.")
    else:
        print(f"{NUM} Is Not A Strong Number.")
Strong(145)
Strong(265)